testing/test_terminal.py

```diff
@@ -428,12 +428,14 @@ def test_long_xfail():
             + [
                 (
                     "test_verbose_skip_reason.py::test_long_skip SKIPPED"
-                    " (cannot do foobar because baz is missing due to I don't know what) *"
+                    " (1 cannot do foobar"
                 ),
+                "because baz is missing due to I don't know what) *",
```

Comment:
@nicoddemus,

Thanks for taking a look.
> It does not look like it was doing what was intended, or am I missing something?

The test `test_verbose_skip_reason` runs pytest twice, once with `-v`, and once with `-vv`.  The output you provided was from the first run, which is less verbose, hence the truncation.  I modified the expected output for the second run to expect the output to continue onto a new line in the middle of the output.  This comment annotates that modification to help show more clearly that the modified test does check for the changed behavior.
Hopefully that makes sense, but let me know if it needs further clarification or changes to make it more clear.  For example, we could break the runs into two separate tests, and move the common inputs and outputs into a fixture rather than the test body.

Reply:
Ahh right, sorry I missed that, thanks for the explanation. 👍